modules/Eliza.py

```diff
@@ -12,6 +12,9 @@ def __init__(self):
         self.psychobabble = psychobabble
         self.reflections = reflections
 
+    def __str__(self) -> str:
+        return "Eliza"
```

Comment:
what is the benefit of explicit types in these situations, please?

Reply:
Not much when it comes to dunder functions, other than it allows static type checkers to check it so if anyone changes it things like mypy can catch it.  Ideall, I'll slowly type all of Stampy, then we can have a mypy CI check.

Reply:
static type checkers like mypy don't complain if `__str__` returns a nonstring without explicit type? also if someone returns a number, they will change the signature to `-> int` - I myself don't really see how that helps prevent bugs,, is the explicit type something that you don't expect your future self to change in case you would change the return value or something?

Reply:
Did some testing, mypy will not flag this:

```py
def __str__(self):
    return 1
```
but Python will throw a type error if you ever call `str(obj)`.

It will flag this:
```py
def __str__(self) -> str:
    return 1
```
as `error: Incompatible return value type (got "int", expected "str")`.

And, it will flag:
```py
def __str__(self) -> int:
    return 1
```
as `error: Return type "int" of "__str__" incompatible with return type "str" in supertype "object"`.

Reply:
nice, thanks for checking and sharing info!